packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel.StubLibrary/src/StubLibraryVisitor.cs

```diff
@@ -16,7 +16,7 @@ internal class StubLibraryVisitor : ScmLibraryVisitor
 
         protected override TypeProvider? Visit(TypeProvider type)
         {
-            if (!type.DeclarationModifiers.HasFlag(TypeSignatureModifiers.Public))
+            if (!type.DeclarationModifiers.HasFlag(TypeSignatureModifiers.Public) && !type.Name.StartsWith("Unknown"))
```

Comment:
would there be a case that our `Unknown` variant is public?
I think the first condition should cover all our cases.
And the second might mistakenly hit the case that there is a normal model starting with `Unknown`, and even worse when a case `Unknown` in that type is not a full word such as `Unknowning` or something, it might not spell properly but it could happen

Reply:
would be pretty rare and the worst case scenario is we get one extra stub class.  I am not worried about that outcome if it ever did happen unless I am missing something